verifier/truststore/errors.go

```diff
@@ -0,0 +1,54 @@
+// Copyright The Notary Project Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package truststore
+
+// ErrorTrustStore is used when accessing specified trust store failed
+type ErrorTrustStore struct {
```

Comment:
I updated the new error types, they are under `[verifier/truststore/errors.go](https://github.com/notaryproject/notation-go/pull/345/files#diff-7f18bd96a3672bb001f5df982e3d300c7b1b35b51423c181a96834a9f07a0470)`.
Kept the original naming (errors in notation-go/errors.go) to avoid any breaking change.